tests/baselines/reference/api/typescript.d.ts

```diff
@@ -4183,14 +4182,11 @@ declare namespace ts {
      * @returns A 'Program' object.
      */
     function createProgram(rootNames: ReadonlyArray<string>, options: CompilerOptions, host?: CompilerHost, oldProgram?: Program, configFileParsingDiagnostics?: ReadonlyArray<Diagnostic>): Program;
-    interface ResolveProjectReferencePathHost {
-        fileExists(fileName: string): boolean;
-    }
     /**
      * Returns the target config filename of a project reference.
      * Note: The file might not exist.
      */
-    function resolveProjectReferencePath(host: ResolveProjectReferencePathHost, ref: ProjectReference): ResolvedConfigFileName;
+    function resolveProjectReferencePath(ref: ProjectReference): ResolvedConfigFileName;
```

Comment:
this is worth documenting as a breaking change. I'll need to update my use of that function or write my own that works for all TypeScript versions I want to support

Reply:
@RyanCavanaugh May be I can just revert the api change and make it such that first parameter is any so we don't break others

Reply:
Maybe you could just add an overload that takes two parameters and discards the first one. This could be marked `@deprecated`